### DISCUSSION TITLE
How to preview a align equation in Visual Studio or Rstudio

### DISCUSSION THREAD

**Discussion by victorfeagins** [quarto-dev/quarto-cli#9417] [Q&A]
### Description

When I am working on more complex equations with algn it can be time consuming having to re render the document to see make sure the equation is correctly updated. Is there a way to preview the align environment? I notice R studio and VS code both will preview latex equations wrapped in dollar signs but to my knowledge the dollar sign notation does not work with algin equations.

````qmd
---
title: "Reproducible Quarto Document"
format: html
---

```{=tex}
\begin{align} 
    \text{Level 1}: Y_{i(j,l)} &= \beta_{0(j,l)}  + R_{i(j,l)}\\
    \text{Level 2}: \beta_{0(j,l)} &= \gamma_{00} + U_{0j} + W_{0l}
\end{align}
```
Previews correctly:
$$
 \text{Level 1}: Y_{i(j,l)} = \beta_{0(j,l)}  + R_{i(j,l)}
$$

$$
 \text{Level 2}: \beta_{0(j,l)} = \gamma_{00} + U_{0j} + W_{0l}
$$
````

**Comment by mcanouil** [comment]
> I notice R studio and VS code both will preview latex equations wrapped in dollar signs but to my knowledge the dollar sign notation does not work with algin equations.

From where does your knowledge come from?

<img width="1624" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/00700ad4-335c-4127-bdaa-283828ab6f76">
